src/Camp.css

```diff
@@ -1,35 +1,171 @@
-.Center {
-  display: block;
-  width: 50%;
-  margin: 70px auto;
+.Wrapper {
+  background-color: white;
 }
 
+.Wrapper h1{
 
-.Card {
+
+}
+
+.resume {
+  display: flex;
+  flex-direction:row;
+  flex-wrap: wrap;
   text-align: center;
-  background-color: #2C7E9A;
-  max-width: 450px;
-  border: 3px solid gray;
-  border-radius: 12px;
+  align-content: space-between;
+  background-color: #292727;
```

Comment:
If you use SCSS you can specify your color palette in a series of variables. 
That way you ensure that your colors are consistent over time.